### ISSUE TITLE
Prop type not appearing in playroom panel if story sets it to false

### ISSUE THREAD

**Issue by ptibbetts** [rbardini/storybook-addon-playroom#7]
Hey,

I have a component that accepts an `isLoading` prop, which I'm using to add a CSS class:

```jsx
let classNames = "collection content";
if (isLoading) {
    classNames = `${classNames} is-loading`; // adds a greyed out filter
}
return (
    <div className={classNames}>
...
```

In my story I am explicitly setting this to false:
```jsx
export const Default = () => (
    <Collection
        type='news'
        collection={collectionData}
        isLoading={false}
    />
)
```

which works fine on the `canvas` and `docs` panels, the prop gets set to false and the CSS class doesn't apply, however inside the playroom tab I'm only getting the following output for the story:

```jsx
<Collection
  collection={collectionData} /* a really big object */
  type="news"    
/>
```

I can edit the playroom story code to include `isLoading={false}` and the CSS class gets removed, but the playroom addon seems to be ignoring this prop because I'm explicitly setting it to false.

I only noticed it because I defaulted `isLoading` to true, which I don't really need to do, but still the storybook addon is acting differently to the story output of other panels.

**Comment by rbardini** [comment]
Hi @ptibbetts!

I think this is caused by `useBooleanShorthandSyntax` being `true` by default in [`react-element-to-jsx-string`](https://github.com/algolia/react-element-to-jsx-string#reactelementtojsxstringreactelement-options), which is used to transform the rendered story into a JSX string. I'll play with the options to see if it solves the issue.

The Docs addon relies on Storybook's [source-loader](https://github.com/storybookjs/storybook/tree/master/lib/source-loader), that's why the code looks slightly different in Playroom. I plan to use the same loader to have more predictable results.

Relates to #5.

**Comment by rbardini** [comment]
Turns out if I disable `useBooleanShorthandSyntax` it works but also converts `isLoading` (without value) to `isLoading={true}`, which is not nice but at least is consistent 🤔

**Comment by rbardini** [comment]
Just released version 1.1.0, which allows defining custom `react-element-to-jsx-string` options. By setting `useBooleanShorthandSyntax` to `false` you should be able to see false boolean props in Playroom.
